docs/installation/docker.md

```diff
@@ -30,6 +30,9 @@ docker build -t linbreux/wikmd:latest .
 version: "2.1"
 services:
   wikmd:
+    build:
```

Comment:
I think we should still try to download the image from dockerhub first.  Maybe we can add an other paragraph like: "run local docker build" or something